indico/modules/designer/templates/template.html

```diff
@@ -288,6 +294,14 @@
                                 class="i-button move-button icon-only text-color subtle icon-expand"
                                 data-direction="bottom">
                         </button>
+                        <button title="{% trans %}Send item to the back{% endtrans %}"
```

Comment:
Let's be consistent on the titles, e.g.:

- Move item to the back
- Move item to the front